modules/phase_field/include/ics/RampIC.h

```diff
@@ -0,0 +1,56 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+
+#ifndef RAMPIC_H
+#define RAMPIC_H
+
+#include "InitialCondition.h"
+#include "InputParameters.h"
+#include "FEProblem.h"
+#include "MooseMesh.h"
+
+#include <string>
+
+//Forward Declarations
+class RampIC;
+class MooseMesh;
+
+template<>
+InputParameters validParams<RampIC>();
+
+/**
+ * Defines a boundary condition that forces the value to be a user specified
```

Comment:
No, it does not. Please make sure that copy pasted comments are updated to reflect what the code actually does.